### PR TITLE
[Middleware_01] Create public UML diagrams

### PR DESCRIPTION
**Pull request by jvalentini555**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,7 @@
 hs_err_pid*
 
 # .DS_store files created by Finder on Mac
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+docs/uml-class-diagrams/middleware01/production/Middleware01Package.png
+docs/uml-class-diagrams/middleware01/production/Middleware01ProductionPackage_DirectDependencies.png
+docs/uml-class-diagrams/middleware01/production/Middleware01ProductionPackage_InternalDependencies.png
```
Comment (andrewdimmer) [inline .gitignore]: Not required to commit, but you can remove this if you want to. The .png files will be being added to the repo eventually, so if they get committed now it's perfectly fine.

**Review by andrewdimmer — DISMISSED** [review]
Looks good to me! 1 minor comment about the .gitignore, but it doesn't need to be taken care of before merging if you don't want to.

**Review by TessaPeruzzi — DISMISSED** [review]
looks good!

**Comment by jvalentini555** [comment]
Just changed the .gitignore, that's all

**Review by andrewdimmer — APPROVED** [review]
Looks good to me!

**Review by TessaPeruzzi — APPROVED** [review]
looks good!

**Pull request merged**